hasher-matcher-actioner/.devcontainer/startup.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 set -e
 export OMM_CONFIG=/workspace/.devcontainer/omm_config.py
-flask --app OpenMediaMatch.app db upgrade --directory /workspace/src/OpenMediaMatch/migrations
+MIGRATION_COMMAND=1 flask --app OpenMediaMatch.app db upgrade --directory /workspace/src/OpenMediaMatch/migrations
```

Comment:
non-blocking question: As an alternative, can we detect whether the "Run" command has been triggered, and only startup apscheduler then?